accelerator/http.c

```diff
@@ -178,7 +178,7 @@ static inline int process_get_ta_info_request(ta_http_t *const http, char **cons
 
 static inline int process_proxy_api_request(ta_http_t *const http, char const *const payload, char **const out) {
   status_t ret;
-  ret = api_proxy_apis(&http->core->iota_service, payload, out);
+  ret = api_proxy_apis(&http->core->ta_conf, &http->core->iota_service, payload, out);
```

Comment:
It is weird to call `api_proxy_apis`. Can you revise?

Reply:
how about `wrapper_api_proxy_apis()`?

Reply:
Let's try `proxy_api_wrapper`.